Rakefile

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 require "bundler/gem_tasks"
-task default: :spec
+
+task default: [:test]
+
+task :test do
```

Comment:
I think I’ve mentioned this before but I’d love to help with testing. Do you think just using the rails action cable test library is enough? Interested in any ideas/guidance here.

Reply:
Let's plan on a pair session sometime soon to kick off this effort.

Reply:
@hopsoft sounds good to me 👍